answer/damegane/JS/06.bored/06.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>bored</title>
+  <style>
+    #img_emotion {
+      width: 70%;
+      margin: 0 auto;
+      display: block;
+    }
+
+    #emotion {
+      margin: auto;
+      padding: 0;
+      color: #0ff;
+      font-size: 40px;
+      font-weight: bold;
+      -webkit-text-stroke: 1px #00f;
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translateY(-50%) translateX(-50%);
+      z-index: 10;
+    }
+  </style>
+</head>
+<body>
+  <div class="container">
+    <p id="emotion" onclick="change();">bored</p>
+    <img src="img/bored.jpg" alt="bored" id="img_emotion">
+  </div>
+  <script>
+    const change = () => {
+      document.getElementById('emotion').innerHTML = 'AWESOME';
```

Comment:
nits.どうさとしては問題なさそうですが、javascirptで `getElxxx`系のDom検索系メソッドを叩く時は一度取得したらそれを使い回す方がパフォーマンス的に good です。

```suggestion
//      document.getElementById('emotion').innerHTML = 'AWESOME';
//      document.getElementById('emotion').style.color = '#ff00ea';
//      document.getElementById('emotion').style.webkitTextStrokeColor = '#f00';
const emotion = document.getElementById('emotion');
emotion.innerHTML = 'AWESOME';
emotion.style.color = '#ff00ea';
emotion.style.webkitTextStrokeColor = '#f00';
```